### PR TITLE
add new changefeed level config 

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflow/pull/9136

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -130,12 +130,18 @@ write-key-threshold = 0
 #    {matcher = ['test6.*'], partition = "ts"}
 # ]
 
-# protocol 用于指定传递到下游的协议格式
-# 当下游类型是 Kafka 时，支持 canal-json、avro 两种协议。
+# protocol 用于指定编码消息时使用的格式协议
+# 当下游类型是 Kafka 时，支持 canal-json、avro、Open-Protocol 。
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:134]:
```suggestion
# 当下游类型是 Kafka 时，支持 canal-json、avro 和 open-protocol。
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -130,12 +130,18 @@ write-key-threshold = 0
...
 # 当下游类型是存储服务时，目前仅支持 canal-json、csv 两种协议。
 # 注意：该参数只有当下游为 Kafka 或存储服务时，才会生效。
 # protocol = "canal-json"
 
+# 指定 Delete 事件的输出内容，只对 canal-json、Open-Protocl 协议有效。
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:139]:
```suggestion
# delete-only-output-handle-key-columns 用于指定 Delete 事件的输出内容，只对 canal-json 和 open-protocol 协议有效。从 v7.2.0 开始引入。
```

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -130,12 +130,18 @@ write-key-threshold = 0
...
+# 默认为 false，输出所有列的内容。当设置为 true，只输出一行的主键列，或唯一索引列的内容。
+# Avro 协议不受该参数控制，总是只输出一行的主键列，或唯一索引列的内容。
+# CSV 协议不受该参数控制，总是输出所有列的内容。
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:142]:
```suggestion
# 默认值为 false，即输出所有列的内容。当设置为 true 时，只输出一行的主键列，或唯一索引列的内容。
# Avro 协议不受该参数控制，总是只输出一行的主键列，或唯一索引列的内容。
# CSV 协议不受该参数控制，总是输出所有列的内容。
```

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 4]
```diff
@@ -130,12 +130,18 @@ write-key-threshold = 0
...
-# protocol 用于指定传递到下游的协议格式
-# 当下游类型是 Kafka 时，支持 canal-json、avro 两种协议。
+# protocol 用于指定编码消息时使用的格式协议
+# 当下游类型是 Kafka 时，支持 canal-json、avro 和 open-protocol。
 # 当下游类型是存储服务时，目前仅支持 canal-json、csv 两种协议。
 # 注意：该参数只有当下游为 Kafka 或存储服务时，才会生效。
 # protocol = "canal-json"
 
+# delete-only-output-handle-key-columns 用于指定 Delete 事件的输出内容，只对 canal-json 和 open-protocol 协议有效。从 v7.2.0 开始引入。
+# 默认值为 false，即输出所有列的内容。当设置为 true 时，只输出一行的主键列，或唯一索引列的内容。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:140]: 请问这里的“一行” 是什么意思呢

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**